### DISCUSSION TITLE
Instead of editing on a new page, is there a way to edit resource row using dialog instead.

### DISCUSSION THREAD

**Discussion by JenuelDev** [laravel/nova-issues#3365] [Q&A]
I think editing a resource by opening a new page might take up time to do it because it has to reload a new page. But is it possible to make it a dialog instead?

**Comment by crynobone** [comment]
Laravel Nova is a SPA, the benefit of opening a new page vs dialog is almost identical.

**Reply by JenuelDev** [reply]
yes but I believe for user experience, the dialog is a lot easier. Going to other pages fetches data every time I press back.
but with modal, the fetched data stays and doesn't need to reload, closing a modal is easy than pressing a back button in new page.
